eng/restore/repoRestore.targets

```diff
@@ -4,7 +4,7 @@
     <RestoreProjectStyle Condition="'$(MSBuildProjectExtension)' != '.depproj'">None</RestoreProjectStyle>
   </PropertyGroup>
 
-  <Import Project="$(RepositoryEngineeringDir)CodeAnalysis.targets" />
+  <Import Project="$(RepositoryEngineeringDir)codeAnalysis.targets" />
```

Comment:
casing?

Reply:
My strategy is to use upper-case for Arcade API inclusions and camelCase for everything else.

Reply:
Ok, I didn't notice the file name change as part of this, wheras I did see a change for analyzers.

Reply:
unfortunately the git client didn't recognize the name changes and I had to do them again now...

Reply:
to be more precise, files which aren't common like DisableSourceControlManagement.targets and Arcade endpoints like Build.props, Tools.props, Versions.props, Signing.props and Version.Details.xml are camelCase.

Reply:
`git mv -f` or `git config core.ignorecase false`  can help.

I like the convention, can we document it somewhere?

Reply:
I will do that in a subsequent PR when I have time. Thanks for the command :)